<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="toolbarStyleLabel">
     <property name="text">
      <string>Toolbar Style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="toolbarStyleComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Icons Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text Alongside Icons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text Under Icons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>System Style</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="oneInstanceCheckBox">
     <property name="text">
      <string>Allow only one instance running</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="hideCheckBox">
     <property name="text">
      <string>Hide window on startup</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="nativeMenuBarCheckBox">
     <property name="toolTip">
      <string>Need to restart the application for this change to take effect</string>
     </property>
     <property name="text">
      <string>Use native menu bar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="startAtLoginCheckbox">
     <property name="text">
      <string>Start at login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
